app/controllers/listings/lease_ups/applications_controller.rb

```diff
@@ -22,7 +21,7 @@ def listing_service
     end
 
     def compacted_listing(full_listing)
-      full_listing.slice('Id', 'Name', 'Building_Street_Address')
+      full_listing.slice('Id', 'Name', 'Building_Street_Address', 'Listing_Lottery_Preferences')
```

Comment:
I just added a field to this too. By the end of next month it'll be all the listing fields 😆

Reply:
hehe